server/models/schema.sql

```diff
@@ -558,6 +558,7 @@ CREATE TABLE `enterprise` (
   `logo`            VARCHAR(100) DEFAULT NULL,
   `currency_id`     TINYINT(3) UNSIGNED NOT NULL,
   `po_box`          VARCHAR(30) DEFAULT NULL,
+  `helpdesk`        VARCHAR(512) DEFAULT NULL,
```

Comment:
Let's make this TEXT.  From experience, I know we'll regret it one day if we don't.

Reply:
Done